api/proto/common/common.proto

```diff
@@ -29,13 +29,20 @@ message BlobCommitment {
 	uint32 data_length = 4;
 }
 
+message BlobHeader {
+	// Blob version
+	uint32 version = 1;
+	repeated uint32 quorum_numbers = 2;
+	common.BlobCommitment commitment = 3;
+	bytes payment_header_hash = 4;
+}
+
 // BlobCertificate is what gets attested by the network
 message BlobCertificate {
-	uint32 version = 1;
-	bytes blob_key = 2;
-	BlobCommitment blob_commitment = 3;
-	repeated uint32 quorum_numbers = 4;
+	bytes blob_key = 1;
+	BlobHeader blob_header = 2;
 	uint32 reference_block_number = 5;
+	repeated string relays = 4;
```

Comment:
We need to decide what form the relay identifier will take. If relays are registered on chain then we can use a short identifier, e.g. a `uint32` or similar. Passing the relay URLs here will be convenient but could bloat the size of the certificate.

Reply:
updated